hpcbench/driver.py

```diff
@@ -50,6 +50,7 @@
 )
 from . campaign import (
     from_file,
+    NodeSet,
```

Comment:
I don't understand why you are importing this from campaign - shouldn't this come from `hpcbench.ext.ClusterShell`?

Reply:
Yes it is the same symbol. My point is that driver is using campaign. And campaign could decide to use another NodeSet implementation in the future, that wouldn't change logic in driver.
Does that make sense to you?

Reply:
Ah, I see. Yes makes sense!